pyfstat/core.py

```diff
@@ -699,6 +718,9 @@ def init_computefstatistic(self):
 
         if self.transientWindowType:
             logging.info("Initialising transient parameters")
+            self.maxTwoF = 0
+            if getattr(self, "BtSG", False):
```

Comment:
```suggestion
            if self.BtSG:
```
I don't recall if there was a good reason for this specific one or we simply forgot to change it.
Attribute `BtSG` is always initialized, so no need for getattr.

Reply:
Put it back in specifically because when called from semicoherent classes, it was not initialised.